appengine/flexible/tasks/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Use the official Python image.
 # https://hub.docker.com/_/python
-FROM python:3.7
+FROM python:3.8
```

Comment:
I think appengine flex technically is only supposed to run 3.6 so I'll manually revert that (https://cloud.google.com/appengine/docs/python/#choose-your-preferred-environment)